subprojects/docs/src/docs/userguide/upgrading_version_4.adoc

```diff
@@ -131,19 +155,38 @@ Tasks & properties::
  * You can no longer register invalid inputs and outputs via link:{javadocPath}/org/gradle/api/tasks/TaskInputs.html[TaskInputs] and link:{javadocPath}/org/gradle/api/tasks/TaskOutputs.html[TaskOutputs].
  * The `TaskDestroyables.file()` and `TaskDestroyables.files()` methods have been removed — use link:{javadocPath}/org/gradle/api/tasks/TaskDestroyables.html#register-java.lang.Object++...++-[TaskDestroyables.register()] instead.
  * `SimpleWorkResult` has been removed — use link:{javadocPath}/org/gradle/api/tasks/WorkResult.html#getDidWork--[WorkResult.didWork].
+ * Overriding default tasks <<#deprecations_4.8,deprecated in 4.8>> now produces an error.
++
+Attempting to replace a built-in task will produce an error similar to the following:
+  > Cannot add task 'wrapper' as a task with that name already exists.
 Scala & Play::
  * Play 2.2 is no longer supported — please upgrade the version of Play you are using.
  * The `ScalaDocOptions.styleSheet` property has been removed — the Scaladoc Ant task in Scala 2.11.8 and later no longer supports this property.
+Kotlin DSL::
+ * Artifact configuration accessors are now typed `NamedDomainObjectProvider<Configuration>` instead of simply `Configuration`
+ * `PluginAware.apply<T>(to)` was renamed `PluginAware.applyTo<T>(target)`.
++
+Both changes could cause script compilation errors.
++
+See the https://github.com/gradle/kotlin-dsl/releases/tag/v1.0-RC13#breaking-changes[Gradle Kotlin DSL release notes] for more information and how to fix builds broken by the changes described above.
```

Comment:
@gradle/kotlin-dsl is this link correct? I just copied it from the release notes.

Reply:
It resolves but will need to be changed before 5.0 GA to whatever Kotlin DSL 1.0.x release

Reply:
Scratch that... it doesn't resolve. This does: https://github.com/gradle/kotlin-dsl/releases/tag/v1.0-RC12#breaking-changes

How does our release notes test miss that... I thought it checked all links...

Reply:
5.0-RC1 will go out with 1.0-RC13, the release notes aren't published yet, they will be Monday, that link is correct. You should see the draft here https://github.com/gradle/kotlin-dsl/releases